tools/autobuild/build-boards.sh

```diff
@@ -55,10 +55,8 @@ function build_board {
     $MICROPY_AUTOBUILD_MAKE BOARD=$board BUILD=$build_dir && copy_artefacts $dest_dir $descr $fw_tag $build_dir $@
     rm -rf $build_dir
 
-    # Query variants from board.json and build them. Ignore the special "IDF3"
-    # variant for ESP32 boards (this allows the downloads page to still have
-    # the idf3 files for older releases that used to be explicitly built).
-    for variant in `cat $board_json | python3 -c "import json,sys; print(' '.join(v for v in json.load(sys.stdin).get('variants', {}).keys() if v != 'IDF3'))"`; do
+    # Query variants from board.json and build them.
+    for variant in `cat $board_json | python3 -c "import json,sys; print(' '.join(v for v in json.load(sys.stdin).get('variants', {}).keys()))"`; do
```

Comment:
This list comprehension is simple enough that the `v for v in` bit can now be removed.

Reply:
Of course; fixed. 🤦🏻‍♂️